site/src/pages/DeploySettingsPage/OAuth2AppsSettingsPage/OAuth2AppsSettingsPageView.tsx

```diff
@@ -77,9 +80,13 @@ const OAuth2AppsSettingsPageView: FC<OAuth2AppsSettingsProps> = ({
             {!isLoading && (!apps || apps?.length === 0) && (
               <TableRow>
                 <TableCell colSpan={999}>
-                  <div css={{ textAlign: "center" }}>
-                    No OAuth2 applications have been configured.
-                  </div>
+                  {error ? (
+                    <ErrorAlert error={error} />
+                  ) : (
+                    <div css={{ textAlign: "center" }}>
+                      No OAuth2 applications have been configured.
+                    </div>
+                  )}
```

Comment:
Haha yeah the alert in a table looks so janky.  I was lazy because it is a case that should not happen.

I did not realize it might be possible to have both apps and an error with React Query!  That is a really good point in favor of rendering them separately.  Will fix.